src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -439,7 +439,7 @@ private static void LoadGenesis(ChainSpecJson chainSpecJson, ChainSpec chainSpec
 
         var requestsEnabled = depositsEnabled || withdrawalRequestsEnabled || consolidationRequestsEnabled;
         if (requestsEnabled)
-            genesisHeader.RequestsRoot = Keccak.EmptyTreeHash; ;
+            genesisHeader.RequestsHash = ExecutionRequestExtensions.CalculateHashFromFlatEncodedRequests(new byte[][] { Array.Empty<byte>(), Array.Empty<byte>(), Array.Empty<byte>() });
```

Comment:
You could put the empty requests hash case into the constants, to make it clear what it is and have it centrally defined.

Reply:
also static `EmptyRequest` with those 3 empty arrays.